deploy/helm/sumologic/values.yaml

```
@@ -4409,6 +4409,134 @@ metadata:
       ## To use maxUnavailable, set minAvailable to null and uncomment the below:
       # maxUnavailable: 1
 
+## Configure optional OpenTelemetry Gateway in Agent mode
+## ref: docs/opentelemetry_collector.md#load-balancing-using-the-gateway
+otelgateway:
+  enabled: false
+  deployment:
+    nodeSelector: {}
+    tolerations: []
+    resources:
+      limits:
+        memory: 2Gi
+        cpu: 1000m
+      requests:
+        memory: 196Mi
+        cpu: 50m
+    ## Add custom labels only to otelagent daemonset.
+    podLabels: {}
+    ## Add custom annotations only to otelagent daemonset.
+    podAnnotations: {}
+    image:
+      repository: public.ecr.aws/sumologic/sumologic-otel-collector
+      tag: 0.0.50-beta.0
+      pullPolicy: IfNotPresent
+    livenessProbe:
+      periodSeconds: 15
+      timeoutSeconds: 10
+      failureThreshold: 3
+    readinessProbe:
+      periodSeconds: 10
+      timeoutSeconds: 3
+      failureThreshold: 3
+    startupProbe:
+      periodSeconds: 10
+      timeoutSeconds: 3
+      failureThreshold: 30
```

I believe we can check this a bit more often during the startup to shorten the time that's necessary to make the instance ready to handle traffic. Perhaps 5 seconds would be better?

At the same time let's increase the `failureThreshold` so that the total time wait time ( `5 * 60 = 300s` ) is a bit longer.

```suggestion
    startupProbe:
      periodSeconds: 5
      timeoutSeconds: 3
      failureThreshold: 60
```